test/modules/fundingManager/bondingCurve/utils/mocks/BondingCurveBaseV1Mock.sol

```diff
@@ -62,6 +62,19 @@ contract BondingCurveBaseV1Mock is BondingCurveBase_v1 {
         returns (uint)
     {}
 
+    //--------------------------------------------------------------------------
```

Comment:
Missing space between `//` and `---`
(maybe just update the whole file if you are at it?)